core/src/main/java/org/elasticsearch/transport/TransportSettings.java

```diff
@@ -19,8 +19,11 @@
 package org.elasticsearch.transport;
 
 import org.elasticsearch.common.settings.Setting;
+import org.elasticsearch.common.settings.Setting.Scope;
 import org.elasticsearch.common.settings.Settings;
 
+import static org.elasticsearch.common.settings.Setting.groupSetting;
```

Comment:
I'm pretty willing to use static imports in general but if the name of the thing is in the method name I'm quite happy with it. If the method were named `group` I'd avoid static imports like the plague.